components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/UIBrandingPreferenceResolver.java

```diff
@@ -2,11 +2,14 @@
 
 import org.wso2.carbon.identity.branding.preference.management.core.exception.BrandingPreferenceMgtException;
 import org.wso2.carbon.identity.branding.preference.management.core.model.BrandingPreference;
+import org.wso2.carbon.identity.branding.preference.management.core.model.CustomText;
 
 /**
  * UI Branding Preference Resolver.
  */
 public interface UIBrandingPreferenceResolver {
 
     BrandingPreference resolveBranding(String type, String name, String locale) throws BrandingPreferenceMgtException;
+    CustomText resolveCustomText(String type, String name, String screen, String locale)
```

Comment:
make these new metho as default interface methods to preserve backward compatibility for already implemented implementations.

Reply:
Adddressed with https://github.com/wso2-extensions/identity-branding-preference-management/commit/fdd3486cfdc51558c723795cf2fb75b6faafcdc5